doc/arkode/guide/source/Usage/ERKStep_c_interface/User_callable.rst

```diff
@@ -1159,36 +1160,56 @@ the code, is provided in :numref:`ARKODE.Mathematics.Adaptivity`.
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Time step error bias factor                               | :c:func:`ERKStepSetErrorBias()`            | 1.5       |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Bounds determining no change in step size                 | :c:func:`ERKStepSetFixedStepBounds()`      | 1.0  1.5  |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Maximum step growth factor on error test fail             | :c:func:`ERKStepSetMaxEFailGrowth()`       | 0.3       |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Maximum first step growth factor                          | :c:func:`ERKStepSetMaxFirstGrowth()`       | 10000.0   |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Maximum allowed general step growth factor                | :c:func:`ERKStepSetMaxGrowth()`            | 20.0      |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Minimum allowed step reduction factor on error test fail  | :c:func:`ERKStepSetMinReduction()`         | 0.1       |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Time step safety factor                                   | :c:func:`ERKStepSetSafetyFactor()`         | 0.96      |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Error fails before MaxEFailGrowth takes effect            | :c:func:`ERKStepSetSmallNumEFails()`       | 2         |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
+   | Explicit stability function                               | :c:func:`ERKStepSetStabilityFn()`          | none      |
+   +-----------------------------------------------------------+--------------------------------------------+-----------+
 
 
 
+.. c:function:: int ERKStepSetAdaptController(void* arkode_mem, SUNAdaptController C)
+
+   Sets a user-supplied time-step controller object.
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the ERKStep memory block.
+      * *C* -- user-supplied time adaptivity controller.  If ``NULL`` then the PID controller will be created (see :numref:`SUNAdaptController.Soderlind`).
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the ERKStep memory is ``NULL``
+      * *ARK_MEM_FAIL* if *C* was ``NULL`` and the PID controller could not be allocated.
```

Comment:
This documentation is correct as-is, since `ERKStepSetAdaptController` calls the underlying `arkSetAdaptController`, that itself defaults to PID.  I've added a note pointing out this discrepancy, including instructions for reverting to the ERKStep default.